### DISCUSSION TITLE
View images and videos in app instead of opening a browser to do so

### DISCUSSION THREAD

**Discussion by Tresquel** [community/community#4358] [Mobile] [answered]
Exactly what the title says

**Comment by taki-on** [comment, marked as answer]
Thank you for your feedback and sharing your idea @Tresquel. I will forward the idea to the product team, so that we can discuss it internally. Thanks again and please keep your feedbacks coming to us!

**Reply by LinqLover** [reply]
Videos are currently not displayed at all in the app, were you also proposing this to the team? Alternatively, IMO videos should at least be displayed as a simple link instead of missing completely in the app.

**Reply by Tresquel** [reply]
"were you also proposing it to the team?"

I don't really know what do you mean by that, I'd just like to be able to open images and videos in-app instead of opening it in a browser.

**Reply by LinqLover** [reply]
I am not sure whether you are referring to the expected behavior when opening an image file/video file from the "Browse code" section or also to the media preview inside a Markdown file. I have noticed that a README.md that contains a `<video>` tag is displayed without any trace of the video in the app.

**Reply by Tresquel** [reply]
Both options should be added.
I didn't know about the `<video>` tag not displaying earlier, maybe it's a bug?
